packages/gridproxy_client/src/builders/contracts.ts

```diff
@@ -13,6 +13,19 @@ export enum ContractState {
   Deleted = "Deleted",
 }
 
+export enum SortByContracts {
+  CreatedAt = "created_at",
+  TwinId = "twin_id",
+  ContractId = "contract_id",
+  Type = "type",
+  State = "state",
+}
+
+export enum SortOrderContracts {
```

Comment:
No need to have contracts in here, ASC and DSC has not relation with the contracts

Reply:
Do you mean it's global for the request itself not specific for the contracts?